spec/requests/schools/ects/change_email_address_wizard_spec.rb

```diff
@@ -0,0 +1,143 @@
+describe "Schools::ECTs::ChangeEmailAddressWizardController" do
+  let(:school) { FactoryBot.create(:school) }
+  let(:teacher) { FactoryBot.create(:teacher) }
+  let(:ect_at_school_period) do
+    FactoryBot.create(
+      :ect_at_school_period,
+      :ongoing,
+      teacher:,
+      school:,
+      email: "ect@example.com"
+    )
+  end
+
+  before do
+    allow(Rails.application.config)
+      .to receive(:enable_schools_interface)
+      .and_return(true)
+  end
```

Comment:
Don't think we need this?

Reply:
The flag is used in a guard clause now we have established multiple policies and a school user role.